### PR TITLE
ARTK-100: rework of readme.txt

### PR DESCRIPTION
**Pull request by ThorstenBux**
mainly correcting broken links

### CODE REVIEW

**Comment by philip-lamb** [comment]
Please rebase from master and push.

**Comment by ThorstenBux** [comment]
Merged

**Comment by ThorstenBux** [comment]
Rebased and ready to merge

File: README (ARToolKit for Android).txt [hunk 1]
```diff
@@ -108,17 +110,17 @@ ARToolKit v5.2 was the first major release under an open source license in sever
...
 
 Please see http://www.artoolkit.org/documentation/Using_automatic_online_camera_calibration_retrieval for details on this feature and the code changes required.
 
 
 libKPM usage.
 -------------
 
-libKPM, which performs key-point matching for NFT page recognition and initialization now use a FREAK detector framework, contributed by DAQRI. Unlike the previous commercial version of libKPM which used SURF features, FREAK is not encumbered by patents. libKPM now joins the other core ARToolKit libraries under an LGPLv3 license. Additionally the new libKPM no longer has dependencies on OpenCV’s FLANN library, which should simply app builds and app distribution on all supported platforms.
+libKPM, which performs key-point matching for NFT page recognition and initialization now use a FREAK detector framework, contributed by DAQRI. Unlike the previous commercial version of libKPM which used SURF features, FREAK is not encumbered by patents. libKPM now joins the other core ARToolKit libraries under an LGPLv3 license. Additionally the new libKPM no longer has dependencies on OpenCV’s FLANN library, which should simplyfy app builds and app distribution on all supported platforms.
```
Comment (BoiseBaked) [inline README (ARToolKit for Android).txt]: I never seen "simplify" spelled with two 'y's. I don't think this is an English spelling of the word. Consider using "simplify" instead.

Comment (ThorstenBux) [inline README (ARToolKit for Android).txt]: Hi @BoiseBaked you are so right, it looked wired while reading but I didn't look it up.
It is fixed now.

**Pull request merged**